Plugins/Json/include/Acts/Plugins/Json/GeometryHierarchyMapJsonConverter.hpp

```diff
@@ -74,30 +74,30 @@ class GeometryHierarchyMapJsonConverter {
     nlohmann::json encoded;
     // only store non-zero identifiers
     if (id.volume()) {
-      encoded["volume"] = id.volume();
```

Comment:
Why did you change the way the `GeometryHierarchyMapJsonConverter` is writing out the geometry tags?

Reply:
As I explain in the description of the PR the key are ordered alphabetically hen writing the file. I used underscore and uppercase  to force the ordering of the key to be :
volume
boundary
layer
approach
sensitive

Reply:
this now fixed with PR #686